docs/networking/port-forwarding.md

```diff
@@ -21,6 +22,14 @@ coder port-forward myworkspace --tcp 8000:8080
 
 For more examples, see `coder port-forward --help`.
 
+## Dashboard
+
+> To enable port forwarding via the dashboard, Coder must be configured with a
+> [wildcard access URL](./admin/configure#wildcard-access-url).
+
+Use the "Port forward" button in the dashboard to access ports
+running on your workspace.
```

Comment:
Hoping to add an image here once the dialog is complete (I also struggled finding a way to test it)

Reply:
Maybe @deansheather can help with that.

Reply:
got it!